canonical/ubuntu.com
# Enable auto renwal on by default

## Done

- Enable auto renewal on by default

## QA

- Go to /pro/dashboard
- Click renewal settings and check if checkbox is checked by default 
<img width="396" alt="image" src="https://user-images.githubusercontent.com/57550290/211035581-307f1e52-91a9-441c-a3fa-9585c15b7cd1.png">

## Issue / Card

Fixes #https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/801?modal=detail&selectedIssue=WD-680
